docs/guides/mongodb/schema-manager/initializing-with-script/index.md

```diff
@@ -0,0 +1,355 @@
+
+
+Let’s save this yaml configuration into `vault.yaml` Then create the above `VaultServer` CR
+
+```bash
+$ kubectl apply -f vault.yaml
+vaultserver.kubevault.com/vault created
+```
+
+### Create Separate Namespace For Schema Manager
+
+In this section, we are going to create a new `Namespace` and we will only allow this namespace for our `Schema Manager`. Below is the YAML of the `Namespace` that we are going to create,
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dev
+  labels:
+    kubernetes.io/metadata.name: dev
+```
+
+Let’s save this yaml configuration into `namespace.yaml`. Then create the above `Namespace`,
+
+```bash
+$ kubectl apply -f namespace.yaml
+namespace/dev created
+```
+
+
+### Script with ConfigMap
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-script
+  namespace: dev
+data:
+  init.js: |-
+    use initdb;
+    db.product.insert({"name" : "KubeDB"});
+```
+
+```bash
+$ kubectl apply -f test-script.yaml 
+configmap/test-script created
+```
+
+
+### Deploy Schema Manager Initialize with Script
+
+Here, we are going to deploy `Schema Manager` with the new Namespace that we have created above. Let’s deploy it using this following yaml,
+
+```yaml
+apiVersion: schema.kubedb.com/v1alpha1
+kind: MongoDBDatabase
+metadata:
+  name: sample-script
+  namespace: dev
+  labels:
+    "schema.kubedb.com": "mongo"
+spec:
+  database:
+    serverRef:
+      name: mongodb
+      namespace: demo
+    config:
+      name: initdb
+  vaultRef:
+    name: vault
+    namespace: demo
+  accessPolicy:
+    subjects:
+      - name: "saname"
+        namespace: dev
+        kind: "ServiceAccount"
+        apiGroup: ""
+    defaultTTL: "5m"
+    maxTTL: "200h"
+  init:
+    initialized: false
+    script:
+      scriptPath: "/etc/config"
+      configMap:
+        name: "test-script"
+      podTemplate:
+        spec:
+          containers:
+            - env:
+              - name: "HAVE_A_TRY"
+                value: "whoo! It works"
+              name: cnt
+              image: nginx
+              command:
+               - /bin/sh
+               - -c
+              args:
+               - ls
+  deletionPolicy: Delete
+```
+Here,
+
+- `spec.database` is a required field specifying the database server reference and the desired database configuration.
+- `spec.vaultRef` is a required field that specifies which KubeVault server to use for user management.
+- `spec.accessPolicy` is a required field that specifies the access permissions like which service account or cluster user have the access and also for how long they can access through it.
+- `spec.init` is an optional field, containing the information of a script or a snapshot using which the database should be initialized during creation.
```

Comment:
Tell something about spec.init.script & its sub-fields, because that is the main focus of this doc.